translations/fr.po

```diff
@@ -749,7 +749,8 @@ msgstr "abandonner"
 
 #: File "src/app/learnocaml_exercise_main.ml", lines 226-227, characters 35-65
 msgid "Grading is taking a lot of time, maybe your code is looping? "
-msgstr "La notation prend du temps, peut-être une boucle infinie dans votre code ? "
+msgstr ""
```

Comment:
I do not understand this change.

Reply:
Good catch, thanks @yurug 
(maybe just a spurious newline introduced by emacs' `po-mode`, but I'll amend my commit to remove it)